pkgs/cosmic-greeter/package.nix

```diff
@@ -43,7 +44,7 @@ rustPlatform.buildRustPackage {
   };
 
   nativeBuildInputs = [ libcosmicAppHook rustPlatform.bindgenHook cmake just ];
-  buildInputs = [ linux-pam ];
+  buildInputs = [ linux-pam systemd ];
```

Comment:
Can you add `udev` instead here if it's only linking `libudev`?

Thank you for this!